pkg/operations/kubernetesupgrade/upgradecluster_test.go

```diff
@@ -61,7 +61,7 @@ var _ = Describe("Upgrade Kubernetes cluster tests", func() {
 	})
 
 	It("Should succeed when cluster VMs are missing expected tags during upgrade operation", func() {
-		cs := api.CreateMockContainerService("testcluster", "1.18.15", 1, 1, false)
+		cs := api.CreateMockContainerService("testcluster", "", 1, 1, false)
```

Comment:
100% of UT should not care about our maintaining the versions support list, so shout if this ever bites you in the future!

cc @mboersma